### ISSUE TITLE
Time Delay for Plot Debris Removal

### ISSUE THREAD

**Issue by cmdbris** [jwlebert/mfgjs2023#7]
After dragging the removal item on the plot, a small circle timer will begin. After around 10 seconds, the layer of debris will be removed. If you upgrade the removal item, the time needed to remove a debris layer will be shortened.

**Comment by jwlebert** [comment]
I think it would also be a good idea if we were to make it cost less each time to remove debris as you upgrade the shovel.

For the timer, I see two possible approaches, either a timer or (like you said) a circle. If we go for the circle, [this video](https://www.youtube.com/watch?v=AyuQXfgVk3U) will likely prove useful.

**Comment by jwlebert** [comment]
Maybe we could do something like this for the timer?

![Shovel Cooldown](https://github.com/jwlebert/mfgjs2023/assets/37757770/05935e64-937f-493e-bf15-8c855dda2a61)

**Comment by cmdbris** [comment]
Good idea, the timers should be over each individual crop rather than in the top corners to emphasize which specific plot is being worked on
